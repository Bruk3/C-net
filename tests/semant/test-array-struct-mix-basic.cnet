
struct mstruct {
	float id;
};

int main(){

	int x;
	while(1) {
		struct mstruct[][] x;

		x[0][0].id;
	}

	return 0;
}
