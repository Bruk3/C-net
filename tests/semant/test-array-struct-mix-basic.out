struct mstruct {
	float id;
};

int main()
{
int x;
while ((int : 1)) {
struct mstruct[][] x;
(float : x[0][0].id);
}
return (int : 0);
}

